applications/StructuralMechanicsApplication/custom_python/add_custom_strategies_to_python.cpp

```diff
@@ -109,7 +109,7 @@ void  AddCustomStrategiesToPython(pybind11::module& m)
 
     // Prebuckling Strategy
     py::class_< PrebucklingStrategyType, typename PrebucklingStrategyType::Pointer,BaseSolvingStrategyType >(m,"PrebucklingStrategy")
-        .def(py::init<ModelPart&, BaseSchemeType::Pointer, BuilderAndSolverPointer, BuilderAndSolverPointer, ConvergenceCriteriaPointer, int, double, double, double, double >())
+        .def(py::init<ModelPart&, BaseSchemeType::Pointer, BuilderAndSolverPointer, BuilderAndSolverPointer, ConvergenceCriteriaPointer, int, double, double, double, double, bool >())
```

Comment:
Please, instead of overloaded constructors use Parameters as input

Reply:
Sry, I don't understand. I only have one single constructor. Do you mean to pass all buckling settings as one parameter variable, anyway?

Reply:
I mean that instead of adding more and more parameters (which is not flexible, and eventually you may even need more), to pass directly a Parameters object as argument